Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java

```diff
@@ -144,26 +148,33 @@ public boolean leaveSession(Player player, TeamColour colour, Session session) {
         LOGGER.log(Level.INFO, "Player {0} left the Session.", player.getUsername());
         return true;
     }
-    
+
     public void leaveSession(Player player, Session session) {
         Room room = player.getCurrentSession().getRoom();
-        
+
         Team teamBlue = room.getTeamBlue();
-        
+
         TeamColour colour = teamBlue.getPlayers().stream().filter(p -> p.getUsername().equals(player.getUsername())).count() > 0 ? TeamColour.BLUE : TeamColour.RED;
         leaveSession(player, colour, session);
     }
-    
+
     public void startGame(Session session) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)